app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
According to the checklist, avoid comparing a variable to `0` if there is no need. Here, you can just check `if masks_to_buy:` instead of `if masks_to_buy != 0:`.